app/scripts/app.js

```diff
@@ -47,7 +47,8 @@ app.config(function($locationProvider, $stateProvider, $urlRouterProvider, confi
                     var filters = {
                         keyword: $stateParams.keyword,
                         organization: $stateParams.organization,
-                        state: $stateParams.state
+                        state: $stateParams.state,
+                        status: $stateParams.status
```

Comment:
Any way to use something like “result” rather than “status” which conflicts with “state”?  :confused: